go/vt/vtgate/executor_test.go

```diff
@@ -496,21 +497,34 @@ func TestExecutorShowColumns(t *testing.T) {
 
 }
 
+func sortString(w string) string {
+	s := strings.Split(w, "")
+	sort.Strings(s)
+	return strings.Join(s, "")
+}
+
+func assertMatchesNoOrder(t *testing.T, expected, got string) {
+	t.Helper()
+	if sortString(expected) != sortString(got) {
+		t.Errorf("for query: expected \n%s \nbut actual \n%s", expected, got)
+	}
+}
```

Comment:
You can use the `AssertMatchesNoOrder` method in the `vitess.io/vitess/go/test/endtoend/utils` package

Reply:
not really. that method expects a `*mysql.Conn`, and I don't have one